src/components/shoppingListGrouping/shoppingListGrouping.module.css

```diff
@@ -4,6 +4,7 @@
 
 .noLists {
   font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  margin-top: -4px;
```

Comment:
Makes text appear to be the same distance from the element above it as that element is from the element above it. (Actually the distance is not the same but text height was making the same distance look unequal.)